user_guide/user_guide.md

```diff
@@ -250,9 +250,115 @@ In the editing mode you can also delete existing Functions.
 
 ### APIs
+*Click the "Manage APIs" button.*
 
 #### Create
+*Click the "Create a new API" button.*
+
+<br/>
+<kbd>
+<img src="../resources/images/create_api1.png" width="450">
+</kbd>
+<br/>
+
+*Click on the switch next to "In protected mode" to make the fields editable.*
+
+We now see a number of fields that can or have to be filled according to the API call.
+
+**Belongs to Function** is the Function that the API implements and can deliver a value for. Use the dropdown menu to select an existing Function.
+
+**API Name** is the name and unique identifier of the API.
+
+**Priority** specifies which API is used first for this Function. There are four different types of priority values: Preferred, High, Medium, Low. Preferred is the highest priority and can only be set onced. If set a second time, the former preferred API's priority will be downgraded.
+
+**Placeholders** are links to other values, namely Fields of the corresponding Function or results of other Functions. The Placeholder can be used in other parts of the API definition like Headers, Parameters or Body. You can use the Placeholders by inserting "§id§" where "id" is the id of the Placeholder. The string gets then replaced by the value of the placeholder i.e. field. You can *click the "Add a new placeholder", bin and "Delete all Placeholders" buttons to add and remove Placeholders.*
+
+<br/>
+<kbd>
+<img src="../resources/images/create_api2.png" width="350">
+</kbd>
+<br/>
+
+**Placeholder ID** gets defined automatically and is needed when using the placeholder in other fields.
+
+**Placeholder Field** specifies the field of the Function which is used to replace the placeholder. You can select in a dropdown menu every field of the Function you defined in the "Belongs to Function" field.
+
+
+Alternatively to replacing the Placeholder by a Function field it can also be replaced by the result of another Function. *Check the "Replace Placeholder by value of the function this API belongs to." box to switch and click again to switch back.*
+
+<br/>
+<kbd>
+<img src="../resources/images/create_api3.png" width="350">
+</kbd>
+<br/>
+
+**Placeholder Function name** specifies the Function the Placeholder receives the result from.
+
+**Placeholder Function Fields** can be set if the Function needs some fields to return the value.
+
+Some API need the values in a String format or in a number format. *Uncheck the "Do not convert placeholder value to 'String' before insertion into a JSON template." checkbox to convert the Placeholder value to a string, check to leave the orginal value.*
+
+You can add **Headers** to the API request.
+*Click the "Add a new key-value pair", bin and "Delete all key-value pairs" buttons to add and remove header fields*
+
+<br/>
+<kbd>
+<img src="../resources/images/create_api4.png" width="350">
+</kbd>
+<br/>
+
+In the "Provide a template for required URL parameter" field you can add **Parameters** in the JSON format to the API request.
```

Comment:
"Provide a template for required URL parameter" -> "Template for the request (URL) Parameters"

Reply:
Noch hinzufügen:
You can also insert parameters directly in the URL when specifying the "API Endpoint URL" field.

Reply:
Name des Eingabefeldes bitte fett, so wie bei den anderen Eingabefeldern.